bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -253,8 +264,6 @@ package org.bigbluebutton.modules.whiteboard.views {
 		public function presenterChange(amIPresenter:Boolean, presenterId:String):void {
 			canvasModel.presenterChange(amIPresenter, presenterId);
 			canvasDisplayModel.presenterChange(amIPresenter, presenterId);
-			whiteboardToolbar.presenterChange(amIPresenter);

```

Comment:
Are the toolbars being notified in some other way about presenter changes? I'm pretty sure these two calls still need to happen.